### PR TITLE
Adding wrapper integration

### PR DESCRIPTION
**Pull request by justinjmoses**

### CODE REVIEW

File: content/integrations/current-integrations.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Current Integrations
 
-Here is a list of Synthetix integrations both on and offchain. For projects wishing to integrate with Synthetix, please see the technical [Integrations guide](/integrations/index/).
+Here is a list of Synthetix integrations both on and offchain. For projects wishing to integrate with Synthetix, please see the technical [Integrations guide](/integrations).
```
Comment (justinjmoses) [inline content/integrations/current-integrations.md:3]: Driveby fix.

File: mkdocs.yml [hunk 2]
```diff
@@ -28,21 +28,22 @@ nav:
       - Releases: releases.md
       - Integrations:
           - Overview: integrations/index.md
-          - Trading:
```
Comment (justinjmoses) [inline mkdocs.yml:31]: I sorted these alphabetically

**Review by liamzebedee — APPROVED** [review]
Hey, this is great. Much more concise than how I might've written it. Two ideas:

* mentioning approvals: we need to approve WETH for mint on the EtherWrapper/NativeEtherWrapper, and sETH for the NativeEtherWrapper (so it can transfer to the base)
* pointing devs to the events in SystemSettings, when the wrapper's settings are updated.

```sol
    event EtherWrapperMaxETHUpdated(uint maxETH);
    event EtherWrapperMintFeeRateUpdated(uint rate);
    event EtherWrapperBurnFeeRateUpdated(uint rate);
```

File: content/integrations/ether-wrapper.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+              synthetixResolver = _snxResolver;
+          }
+
+          function mintsETH() payable external {
+            IEtherWrapper nativeEtherWrapper = synthetixResolver.requireAndGetAddress("NativeEtherWrapper", "NativeEtherWrapper is missing from Synthetix resolver");
+
+            IEtherWrapper etherWrapper = synthetixResolver.requireAndGetAddress("EtherWrapper", "EtherWrapper is missing from Synthetix resolver");
+
+            // this will be checked in the transaction below, but added here as an example
+            require(etherWrapper.capacity >= msg.value, "There isn't enough capacity left for this mint");
```
Comment (liamzebedee) [inline content/integrations/ether-wrapper.md:176]: `capacity()`

**Comment by liamzebedee** [comment]
@justinjmoses wanna take another look?

File: content/governance.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ The Synthetix DAO is the DAO responsible for managing the Synthetix Protocol tre
 
 ### Ambassador DAO (aDAO)
 
-Ambassadors are nominated community members who are appointed as a signer on a Gnosis (multisig)[https://gnosis-safe.io/app/#/safes/0x46abFE1C972fCa43766d6aD70E1c1Df72F4Bb4d], their role is to gather support from Synthetix community members who delegate their governance tokens (from other external protocols) to the multisig, effectively empowering the Ambassadors to participate in external protocols governance systems which have a positive impact on the Synthetix ecosystem (i.e interoperable protocol features, listing of synthetix synths on DEX’s).
+Ambassadors are nominated community members who are appointed as a signer on a Gnosis [multisig](https://gnosis-safe.io/app/#/safes/0x46abFE1C972fCa43766d6aD70E1c1Df72F4Bb4d), their role is to gather support from Synthetix community members who delegate their governance tokens (from other external protocols) to the multisig, effectively empowering the Ambassadors to participate in external protocols governance systems which have a positive impact on the Synthetix ecosystem (i.e interoperable protocol features, listing of synthetix synths on DEX’s).
```
Comment (justinjmoses) [inline content/governance.md:45]: :100:

**Comment by justinjmoses** [comment]
LGTM (though some of those tables have extra columns for some reason?)
